main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2294,11 +2297,6 @@ protected Void doInBackground(final Void... params) {
                 DataStore.saveCache(cache, EnumSet.of(SaveFlag.DB));
                 return null;
             }
-
-            @Override
-            protected void onPostExecute(final Void v) {
-                notifyDataSetChanged();
-            }
```

Comment:
To "fix" the problem that the description page jumps back to the top after closing the dialog.

The reason for the jump is that the page is recreated with the new data. I assume that the content of the page won't be updated without `notifyDataSetChanged()`.